35 post admin-institutions spec added

ValterAro opened the pull request:
New OpenAPI 3.0.3 spec endpoint.
Added POST operation for the admin/institutions path in administrative.yml. Fixed minor errors.
issue #35

guypritchard reviewed (approved):
Worth a demo in tomorrow's stand up - and possibly some documentation in the CI/CD docs.

ValterAro committed:
Merge pull request #85 from buerokratt/main

Merging the updates from the main branch

ValterAro committed:
35 updated OpenAPI spec

Added POST operation to the /admin/institutions path and fixed minor errors in operationId and tag
Issue #35

ValterAro committed:
Update administrative.yml

Fixed $ref and description for participants

ValterAro reviewed (commented):
All good and checked
